chain/indexer/integrated/processor/state.go

```diff
@@ -369,15 +391,15 @@ func (sp *StateProcessor) startActor(ctx context.Context, current, executed *typ
 					sp.pwg.Done()
 				}()
 
+				if err := sp.skipProcessing(changes); err != nil {
```

Comment:
the actor filtering happens in `p.ProcessActors`. Should we do the check there?

Reply:
sure, let's move it.